it/controllers/EditPageISpec.scala

```diff
@@ -16,68 +16,69 @@ class EditPageISpec extends IntegrationSpecBase {
-      Option(document.getElementById("countryCode")).isDefined shouldBe false
-
-      document.getElementById("line1").`val` shouldBe "1 High Street"
-      document.getElementById("line2").`val` shouldBe "Line 2"
-      document.getElementById("line3").`val` shouldBe "Line 3"
-      document.getElementById("town").`val` shouldBe "Telford"
-      document.getElementById("postcode").`val` shouldBe "AB11 1AB"
-
-
-      labelForFieldsMatch(res, idOfFieldExpectedLabelTextForFieldMapping = Map(
-        "line1" -> "Address line 1",
-        "line2" -> "Address line 2",
-        "line3" -> "Address line 3",
-        "town" -> "Town/city",
-        "postcode" -> "UK postcode (optional)"
-      ))
-    }
-    "return UK edit page if no uk parameter provided AND UK mode is false" in {
-      stubKeystore(testJourneyId, testConfigWithAddressNotUkModeAsJson, OK)
-
-      val fResponse = buildClientLookupAddress(path = "edit")
-        .withHeaders(HeaderNames.COOKIE -> sessionCookieWithCSRF, "Csrf-Token" -> "nocheck")
-        .get()
-      val res = await(fResponse)
-
-      res.status shouldBe OK
-      val document = Jsoup.parse(res.body)
-      testElementExists(res, EditPage.nonUkEditId)
-      document.title() shouldBe "Enter the address"
-      document.getElementById("pageHeading").text() shouldBe "Enter the address"
-      document.getElementById("continue").text() shouldBe "Continue"
-
-      document.getElementById("line1").`val` shouldBe "1 High Street"
-      document.getElementById("line2").`val` shouldBe "Line 2"
-      document.getElementById("line3").`val` shouldBe "Line 3"
-      document.getElementById("town").`val` shouldBe "Telford"
-      document.getElementById("postcode").`val` shouldBe "AB11 1AB"
-
-
-      labelForFieldsMatch(res, idOfFieldExpectedLabelTextForFieldMapping = Map(
-              "line1" -> "Address line 1",
-              "line2" -> "Address line 2",
-              "line3" -> "Address line 3",
-              "town" -> "Town/city",
-              "postcode" -> "Postal code (optional)",
-              "countryCode" -> "Country"
-      ))
-    }
+      "return UK edit page if uk param is true AND UK mode is false" in {
+        stubKeystore(testJourneyId, testConfigWithAddressNotUkModeAsJson, OK)
+
+        val fResponse = buildClientLookupAddress(path = "edit?uk=true")
+          .withHeaders(HeaderNames.COOKIE -> sessionCookieWithCSRF, "Csrf-Token" -> "nocheck")
+          .get()
+        val res = await(fResponse)
+
+        res.status shouldBe OK
+        val document = Jsoup.parse(res.body)
+        testElementExists(res, EditPage.ukEditId)
+        document.title() shouldBe "Enter the address"
+        document.getElementById("pageHeading").text() shouldBe "Enter the address"
+        document.getElementById("continue").text() shouldBe "Continue"
+        Option(document.getElementById("countryCode")).isDefined shouldBe false
+
+        document.getElementById("line1").`val` shouldBe "1 High Street"
```

Comment:
There's now an Address helper to make this clearer. See AddressLookupConfirmPageISpec for an example of how to use it.